docs/content/stable/yugabyte-platform/manage-deployments/xcluster-replication/bidirectional-replication.md

```diff
@@ -90,11 +90,9 @@ For index tables, follow the steps in [Add index table](#add-index-table).
 
 ### Add index table
 
+New YSQL indexes are automatically added to xCluster replication if the YSQL table being indexed is bidirectionally replicated. Adding new indexes is supported even if the table being indexed contains data and is actively receiving writes on both the universes.
+
 To add an index table to a bidirectional replication, do the following:
 
-1. Stop the workload on the main table.
-1. Wait for the replication lag for the corresponding main table to become 0.
 1. Create the index table on both universes.
```

Comment:
Creating the index on both sides at the same time is mandatory. If they only creates on one side of will timeout